src/FluentlyHttpClient/FluentHttpResponse.cs

```diff
@@ -111,4 +111,16 @@ public FluentHttpResponse(HttpResponseMessage message, IDictionary<object, objec
 			Items = items ?? new Dictionary<object, object>();
 		}
 	}
+
+	/// <summary>
+	/// Fluent HTTP response, which wraps the <see cref="HttpResponseMessage"/> and adds GraphQL data
+	/// </summary>
+	/// <typeparam name="T"></typeparam>
+	public class GqlResponse<T>
+	{
+		/// <summary>
+		/// GraphQL Content data.
+		/// </summary>
+		public T Data { get; set; }
```

Comment:
ideally this shouldnt have a **set;** exposed, so no one can change the result but the creator. best option is to pass it to the constructor, set as readonly and private setter